sdk/extensions/Microsoft.Azure.WebJobs.Extensions.Clients/samples/local.settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "IsEncrypted": false,
+    "Values": {
+        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+        "MyStorageConnection": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;",
```

Comment:
This is interesting. While this file is JSON it doesn't support nesting configuration as ASP.NET Core does. Filed https://github.com/Azure/azure-sdk-for-net/issues/14826 to track